### PR TITLE
bump to 19 and avoid packaged dependencies

### PR DESCRIPTION
**Pull request by andreaTP**

### CODE REVIEW

File: src/main/java/io/github/evacchi/typed/examples/VendingMachine.java [hunk 1]
```diff
@@ -81,7 +81,7 @@ void vendProduct(String product) {
     }
 
     void releaseChange(int change) {
-        out.printf("CHANGE: %s\n", %d);
```
Comment (andreaTP) [inline src/main/java/io/github/evacchi/typed/examples/VendingMachine.java:84]: How this was working?

File: jitpack.yml [hunk 2]
```diff
@@ -1,10 +0,0 @@
-jdk:
```
Comment (evacchi) [inline jitpack.yml]: I think by deleting this we may break the blog post

File: src/main/java/io/github/evacchi/typed/examples/VendingMachine.java (hunk 1 above)
Comment (evacchi) [reply src/main/java/io/github/evacchi/typed/examples/VendingMachine.java:84]: probably not part of any build

File: jitpack.yml (hunk 2 above)
Comment (andreaTP) [reply jitpack.yml]: restored

**Comment by andreaTP** [comment]
targeting branch `19` so that is safe to merge also.

**Pull request merged**